app/controllers/carts_controller.rb

```diff
@@ -3,12 +3,18 @@ def index
    @cart_items = session[:cart]
     @ordered_foods = {}
     @total = 0
-    @cart_items.each do |food_id, qty|
-      food = Food.find(food_id)
-      @ordered_foods[food_id] = { :food => food , :qty => qty }
-      check_food_status(food, qty, food_id)
+    @pickup_time = 0
+    @prep_total = 0
+    
+    @cart_items.each do |food_id, qty, prep_time|
```

Comment:
Inconsistent indentation detected.<br>Favor `unless` over `if` for negative conditions.